staff/victor-abadia/array-functions/fill.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Be careful!! You shouldn't have here that files! They are from another branch, right??

Reply:
AGREE. REDO the pull request without adding files that do NOT correspond to this challenge.